js/huobi.js

```diff
@@ -4752,41 +4773,17 @@ module.exports = class huobi extends Exchange {
         //     }
         //
         const data = this.safeValue (response, 'data', []);
-        const parsed = this.parseDepositAddresses (data, [ code ], false);
-        return this.indexBy (parsed, 'network');
```

Comment:
we have multiple network addresses , conflicting with each other and returning just one.